### PR TITLE
Update jesus profile

### PR DESCRIPTION
**Pull request by OlushesiToheeb**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_members/lidor-ettinger.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ name: Lidor Ettinger
 position: Contributor
 image_path: /assets/images/community/members/lidor-ettinger.jpeg
 twitter: EttingerLidor
+linkedin: lidor-ettinger-a4725b134
```
Comment (leecalcote) [inline collections/_members/lidor-ettinger.md:7]: Looks good. 👍

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby '2.7.0'
+ruby '2.7.2'
```
Comment (leecalcote) [inline Gemfile:3]: Heads-up - this change affects all contributors' environments. Good to move forward, but might cause a ripple for some.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (leecalcote) [inline package-lock.json:1]: Do we need this file in the Jekyll site?

**Comment by Jashpatel1** [comment]
@OlushesiToheeb the LinkedIn profile changes are to be made in layer5-ng website(i.e., layer5-ng branch).

File: Gemfile (hunk 2 above)
Comment (OlushesiToheeb) [reply Gemfile:3]: I'll just remove it

**Comment by OlushesiToheeb** [comment]
@Jashpatel1 I'll do that also Thank

**Comment by OlushesiToheeb** [comment]
image
@leecalcote which do you suggest i download? because my code is checking for the version here ... 
image

**Review by leecalcote — CHANGES_REQUESTED** [review]
@OlushesiToheeb ideally, this PR has only a single file being updated. Will you double-check on your fork / clone and ensure that they are synced with the latest changes from the upstream repo (this one)?

**Comment by OlushesiToheeb** [comment]
image
everything looks on here @leecalcote

**Pull request closed without merging**